docs/usage_guide.rst

```diff
@@ -887,6 +939,21 @@ You can force a snapshot taking by clicking in the floppy disk icon at the top b
 
 Snapshots won't be saved if there is any error querying information for external sources. If you want to force the snapshot to be saved when an external source is reporting an error you can select the option `Ignore Errors`.
 
+It is possible to remove snapshots from the history in case the information is not correct or needs to be modified. To do it click in their point at the dashboard's graph and this will open a menu
```

Comment:
Also change title of section from `Snapshots` to `Balance Snapshots`

```suggestion
It is possible to remove balance snapshots from the history in case the information is not correct or needs to be modified. To do so click on a snapshot's point at the dashboard's net value graph and this will open a menu.
```